README.md

```diff
@@ -29,6 +29,9 @@ I originally aimed for a no-JS implementation but there was no way of adding ran
 
 Yep, do whatever you want with it.
 
+If you want you can add this badge to help other know about this cool project:  
+[![Using Virtual Bookshelf](https://img.shields.io/badge/Extended%20with-Virtual%20Bookshelf-blue?logo=github)](https://github.com/petargyurov/virtual-bookshelf)
+
```

Comment:
Nice 😄 I would probably re-word it slightly. How about this?

> If you found this project useful you can make use of the following badge to spread the word

Reply:
Sounds great, will update.